Cargo.toml

```diff
@@ -59,7 +59,7 @@ features = ["cblas"]
 [dev-dependencies]
 ndarray-rand = "0.13"
 approx = { version = "0.4", default-features = false, features = ["std"] }
-
+thiserror = "1"
```

Comment:
`thiserror` is already in your dependencies

Reply:
whoops 😄